src/components/Card/Colors/colors.module.css

```diff
@@ -0,0 +1,50 @@
+.palette {
+    position: absolute;
+    background: #fff;
+    /* border: 1px solid black; */
```

Comment:
you can remove this commented code if not required.

Reply:
Yup, i missed it. Thank you!